tutorial/source/svi_part_i.ipynb

```diff
@@ -221,19 +236,29 @@
     "    alpha_q, beta_q = torch.exp(log_alpha_q), torch.exp(log_beta_q)\n",
     "    # sample latent_fairness from the distribution \n",
     "    # Beta(alpha_q, beta_q)\n",
-    "    pyro.sample(\"latent_fairness\", dist.beta, alpha_q, beta_q)  \n",
-    "```\n",
-    "\n",
+    "    pyro.sample(\"latent_fairness\", dist.Beta(alpha_q, beta_q))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
     "There are a few things to note here:\n",
     "\n",
     "- We've taken care that the names of the random variables line up exactly between the model and guide.\n",
     "- `model(data)` and `guide(data)` take the same arguments.\n",
     "- The variational parameters are PyTorch `Variable`s with `requires_grad=True`. If we forget to set the `requires_grad` flag correctly, the parameters won't be trained.\n",
     "- The variational parameters are actually defined in log space. This is because we need `alpha_q` and `beta_q` to be non-negative in order to define a valid beta distribution.\n",
     "\n",
-    "Now we can proceed to do stochastic variational inference. \n",
-    "\n",
-    "```python\n",
+    "Now we can proceed to do stochastic variational inference. "
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
nit: Did you forget to run `make scrub`?

Reply:
i cant because the gmm and gp tutorials have output visualizations baked in as metadata, so it will get wiped if i do. ive been manually `nbstrip`ing